.github/workflows/test.yml

```diff
@@ -0,0 +1,40 @@
+# This is a basic workflow
```

Comment:
With #240 a workflow for Github actions has already been added to the `master` branch. Maybe you could include the caching mechanism into the existing workflow? Although it might actually be better to add the `package-lock.json` itself to the repository which would enable features like dependabot.

Reply:
#240 came after I opened this PR, I can absolutely rebase/merge in master and merge the GitHub actions. I would recommend doing both the caching and the package-lock, would creating a separate PR for the package-lock be better?

Reply:
Oh, good point, I somehow thought this would be alternative to the creating a `package-lock.json`. I think doing both sounds good :) So far, however, the `package-lock.json` has been excluded/explicitly disabled in #225. @Apollon77 @JsonMa What do you think about adding it (back?) to the repo?

Reply:
@JKRhb I took the caching from my test.yml and added it to the npm-test.yml

Reply:
Great, thank you :)